server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -230,7 +230,8 @@ public void createPendingSegmentsTable(final String tableName)
                 + "  UNIQUE (sequence_name_prev_id_sha1)\n"
                 + ")",
                 tableName, getPayloadType(), getQuoteString()
-            )
+            ),
+            StringUtils.format("CREATE INDEX idx_%1$s_sequence_name ON %1$s(sequence_name)", tableName)
```

Comment:
- Could you use simpler syntax `%s` and two arguments: `tableName, tableName`?
 - Please add a comment in code, explaining why this index is needed

Reply:
@yunjing could you fix?